src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -112,6 +113,17 @@ PileupElement getElementForRead(final GATKRead read) {
         return reads.stream().map(r -> PileupElement.createPileupForReadAndOffset(r, offset)).collect(Collectors.toList());
     }
 
+    /**
+     * Helper routine for converting reads and a single offset to a PileupElement list.
+     */
+    public static List<PileupElement> locToReadsPileup(final List<GATKRead> reads, final Locatable loc) {
```

Comment:
Why do we want this instead of using one of the constructors to create a ReadsPileup

Reply:
Because those mostly exist for creating a pileup based on the read elements (eg, the 6th base of these reads) as opposed to this usage which is based on the genomic loc (eg, the base for every read that covers position 1:1000). The second use is what this method is for.